components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngine.kt

```diff
@@ -256,6 +256,7 @@ class GeckoEngine(
     /**
      * See [Engine.registerWebExtensionDelegate].
      */
+    @Suppress("Deprecation")
```

Comment:
nit: Should we create an issue to keep track of when this is going to be addressed? e.g         `runtime.webExtensionController.tabDelegate = tabsDelegate`

Reply:
@Amejia481 nope, see my comment above and the linked issue which will address this in this Sprint :)

Reply:
(maybe even today) :)

Reply:
Awesome 👍